src/python_testing/TC_BRBINFO_4_1.py

```diff
@@ -57,7 +57,8 @@ def desc_TC_BRBINFO_4_1(self) -> str:
 
     def steps_TC_BRBINFO_4_1(self) -> list[TestStep]:
         steps = [
-            TestStep("0", "Preconditions"),
+            TestStep("0",  "DUT commissioned", is_commissioning=True),
```

Comment:
I think the Precondisions step can have the `is_commissioning=True`

Reply:
I thought this would be safer/consistent. Could we keep it like this for now as it works?